modules/uploads/db/schema.prisma

```diff
@@ -0,0 +1,33 @@
+// Do not modify this `datasource` block
+datasource db {
+	provider = "postgresql"
+	url      = env("DATABASE_URL")
+}
+
+model Upload {
+	id						String		@id @default(uuid()) @db.Uuid
+	userId					String?		@db.Uuid
+
+	bucket					String
+	contentLength			BigInt
+
+	createdAt				DateTime	@default(now())
+	updatedAt				DateTime	@updatedAt
+	completedAt				DateTime?
+	deletedAt				DateTime?
+
+	files					Files[]		@relation("Files")
+}
+
+model Files {
```

Comment:
Split this up in to `Files` (with mime, full content length, path) and `FileChunks` (multipart upload id, content length, anything else needed here)

Reply:
Not sure what you mean by this

Reply:
Yes, but this is weird because you're creating a duplicate file for each chunk. So separate the schema about using chunks for uploads vs files for what's actually stored in the db. I know Rivet is designed like what you did, we're doing it better here.

Can chat about this more on Slack.

Reply:
should be called `File`